app/models/health/tasks/notify_care_coordinators_of_patient_eligibility_problems.rb

```diff
@@ -0,0 +1,31 @@
+module Health::Tasks
+  class NotifyCareCoordinatorsOfPatientEligibilityProblems
+    def notify!
+      care_coordinator_ids = Health::Patient.
+        should_be_notified.
+        program_ineligible.
+        distinct.
+        pluck(:care_coordinator_id)
+
+      User.where(id: care_coordinator_ids).each do |user|
+        ineligible_patient_ids = Health::Patient.
+          should_be_notified.
+          no_coverage.
+          where(care_coordinator_id: user.id).
+          pluck(:id)
+
+        no_aco_patient_ids = Health::Patient.
+          should_be_notified.
+          standard_coverage.
+          where(care_coordinator_id: user.id).
+          pluck(:id)
+
+        IneligiblePatientMailer.patients_with_eligibility_problems(user.email, ineligible_patient_ids, no_aco_patient_ids).deliver_now
```

Comment:
Comment out the sending and patient update until we have a chance to test counts on production.